Example/Example/App/Authentication/AuthenticationView.swift

```diff
@@ -23,7 +23,8 @@ class AuthenticationView: UIViewController {
             logoutRedirectUri: "",
             issuer: "",
             redirectUri: "",
-            scopes: "openid profile offline_access email"
+            scopes: "openid profile offline_access email",
+            prompt: nil,
```

Comment:
I wonder if we should just default to `login` instead?

Reply:
yeah, my feeling for mobile is to default to `login`